wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1296,6 +1297,56 @@ def workflow_status_with_date(workflow_state):
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
 
 
+@register.inclusion_tag(
+    "wagtailadmin/shared/keyboard_shortcuts_dialog.html", takes_context=True
+)
+def keyboard_shortcuts_dialog(context):
+    user_agent = context["request"].headers.get("User-Agent", "")
+    is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
+    modifier = "⌘" if is_mac else "Ctrl"
+    shortcuts = {
+        ("common-actions", _("Common Actions")): [
+            {"label": _("Copy"), "shortcut": f"{modifier} + c"},
+            {"label": _("Cut"), "shortcut": f"{modifier} + x"},
+            {"label": _("Paste"), "shortcut": f"{modifier} + v"},
+            {
+                "label": _("Paste without formatting"),
+                "shortcut": f"{modifier} + shift + v",
+            },
+            {"label": _("Undo"), "shortcut": f"{modifier} + z"},
+            {"label": _("Redo"), "shortcut": f"{modifier} + y"},
```

Comment:
The default Redo on Mac is `⌘ + SHIFT + Z`, this one opens up history in Chrome. ;)

Reply:
If we use words for modifier keys like SHIFT, we need to check if these words need translation.
So maybe it is better to stick with icons?

> EN: Press the shift key.
> FR: Appuyez sur la touche Maj.

Reply:
I already checked that with the translations channel, they advised against this.